### ISSUE TITLE
Attempting to remove a user from a class is unsuccessful.

### ISSUE THREAD

**Issue by christianmemije** [learningequality/kolibri#2568]
### Observed behavior

Attempting to remove a user from a class is unsuccessful. Discovered after trying to test #2524, issue existed prior to that PR.

### Expected behavior

Removal of user from class to be successful.

### User-facing consequences

Cannot remove users from class :-1: 

### Errors and logs

```
[06/Nov/2017 15:25:53] "DELETE /api/membership/?user_id=d99f129d62de14896c82836b74133b11&collection_id=e3f9eaeffd31710f90af26efc5350a00 HTTP/1.1" 400 0
```

### Steps to reproduce

* Sign in as superuser admin
* Create a class.
* Enroll user(s).
* Try to remove the user from the class.
* Refresh page to confirm that the user was in fact not removed from the class.

### Context

* Kolibri version: Develop

**Comment by indirectlylit** [comment]
regression - might be using the bulk endpoint to delete a single user

**Comment by rtibbles** [comment]
Fixed in #2695
